### ISSUE TITLE
Latest Licenses not imported

### ISSUE THREAD

**Issue by ostephens** [k-int/KBPlus#24]
The import as of 7am 23rd May does not seem to have included all the license details in /Current/licences (should be 5 licences on the system once these are successfully imported)

**Comment by ianibo** [comment]
2 of the zip files seem to be empty? I'll look at the other 2

**Comment by ianibo** [comment]
For the other 2 I get
Looking up licensor American Institute of Physics
Looking up licensee
ERROR: Unable to lookup licensor or licensee
...
Looking up licensor Cambridge University Press
Looking up licensee
ERROR: Unable to lookup licensor or licensee
Stats: [:]

**Comment by ostephens** [comment]
In the licence I created, I treat "JISC Collections" as the licensee. Liam argued (correctly) this is not accurate as JC doesn't license the stuff, but rather negotiates the licence for others. So Liam left the 'Licensee' blank - which is why the import is failing.

I think Liam is right and we have to represent the different type of relationship but I'd originally thought lets worry about this after the workshop - not realising the import would fail. Obviously we want the licences in there for tomorrow, so think we have 3 choices:

1) Do what I did with the first licence and say JISC Collections is the licensee even though strictly it isn't
2) Allow the import to work even if it doesn't match licensee (just don't create the link)
3) Work out a new relationship and adjust the import to use that

My priority is that people can see the licences in the system, and they are linked to the appropriate Subs Offered - so happy to take recommendations on which of 1, 2 or 3 achieves this with the least work for you
